ts/user-picker/user-picker.multiple.html

```diff
@@ -6,7 +6,7 @@
 	close-on-select="false"
 	reset-search-input="true"
 	uis-open-close="onOpen(isOpen)">
-	<ui-select-match placeholder="{{placeholder}}" allow-clear="allowClear">{{$item.lastName}} {{$item.firstName}}</ui-select-match>
+	<ui-select-match placeholder="{{placeholder}}" allow-clear="{{allowClear}}">{{$item.lastName}} {{$item.firstName}}</ui-select-match>
```

Comment:
if you use `{{!!allowClear}}` here, yo may not have to define it as `scope.allowClear = !!attrs.allowClear ? scope.allowClear : false;`